src/app/components/pagination-component/pagination-component.sass

```
@@ -6,16 +6,18 @@
   display: grid
   justify-content: center
 
-ngb-pagination ::ng-deep .page-item .page-link
-  border: none !important
-  display: flex
-  justify-content: center
-  align-items: center
-  border-radius: 6px
+ngb-pagination
+  &::ng-deep .page-item .page-link
+    border: none !important
+    display: flex
+    justify-content: center
+    align-items: center
+    border-radius: 6px
   & span
     display: none
-ngb-pagination ::ng-deep *
-  font-size: 14px
+    ngb-pagination
+      &::ng-deep *
+        font-size: 14px
```

- I suggest to place `.page-item .page-link` inside the `&::ng-deep`, in cease there will be other styles for pagination.

- Also, yould you please remove parent `&` here `& span`? It's not needed

- Could you please remove `!important`?
I think it should work without it.

`importatn` is kind of last resort, and should be avoided in most of the cases,
usually it could be handled by adding "weight" to selectors - add class/id, or parent, or additional selector in a link

- This looks wrong, It will generate wrong styles, and I think font-size could be placed on the `.page-link`, I assume there will be no other text/inline elements
```
    ngb-pagination
      &::ng-deep *
        font-size: 14px
```   

```suggestion
ngb-pagination
  &::ng-deep
    .page-item .page-link
        border: none
        display: flex
        justify-content: center
        align-items: center
        border-radius: 6px
        font-size: 14px

  span
    display: none
```